pvlib/pvsystem.py

```diff
@@ -2214,6 +2215,12 @@ def sapm(effective_irradiance, temp_cell, module):
         A dict or Series defining the SAPM parameters. See the notes section
         for more details.
 
+    reference_temperature : numeric, optional
+        Reference temperature [C]
```

Comment:
```suggestion
        Reference temperature [°C]
```
I just wanted to show off this page :joy: https://pvlib-python.readthedocs.io/en/stable/contributing/style_guide.html#parameter-names-and-units

Reply:
I tried to stay consistent with the other text in the docstring, but I suppose it's not too far a stretch out of the scope of the PR just to revise all  C -> °C. It needs to be done anyway

Reply:
I (and I'm sure many others) have been lightly bothered for years that we document all these temperature parameters as having units of Colombs...